apps/bookmark/test/marionette/client_options.js

```diff
@@ -2,6 +2,7 @@
 module.exports = {
   settings: {
     'devtools.qps.enabled': false,
-    'language.current': 'en-US'
+    'language.current': 'en-US',
+    'privacy.trackingprotection.shown': true
```

Comment:
There's probably a way to make `privacy.trackingprotection.shown' default to true for integration tests. I think we have to figure out the right way to modify [1]. This way, we don't have to modify so many files, and future system app browser tests don't need to worry about this pop-up. Let's figure out how to do this.

1.) https://github.com/mozilla-b2g/gaia/blob/master/tests/jsmarionette/runner/mozilla-profile-builder/test/fixtures/b2g-profile/settings.json